apps/documentation/src/app/demo-walkthrough/adding-rows/ng-doc.page.ts

```diff
@@ -0,0 +1,11 @@
+import { NgDocPage } from '@ng-doc/core';
+import DemoWalkthroughNgRXCategory from '../ng-doc.category';
+
+const IntroPage: NgDocPage = {
+  title: `Adding Rows`,
+  mdFile: './index.md',
+  order: 9,
+  category: DemoWalkthroughNgRXCategory,
+};
```

Comment:
ng-doc.page.ts and ng-doc.category.ts order properperties need to be unique within a directory. Since these orders live in two different directories, there is no issue here that needs to be corrected.